docs/creating.md

```diff
@@ -95,27 +95,40 @@ suitable to [autoupdate your project safely][the-copier-answersyml-file]:
+| `defaults`         | `bool`                     | When `True`, use default answers to questions.<br>See the [`defaults`][] setting for related information.                                                                                                                                                  |
+| `dst_path`         | `Path`                     | Destination path where to render the subproject.<br>⚠️ When [updating a project][updating-a-project], it may be a temporary directory, as Copier's update algorithm generates fresh copies using the old and new template versions in temporary locations. |
+| `exclude`          | `Sequence[str]`            | Specified additional [file exclusion patterns][patterns-syntax].<br>See the [`exclude`][] setting for related information.                                                                                                                                 |
+| `os`               | `str`                      | The detected operating system, either `"linux"`, `"macos"`, `"windows"` or `None`.                                                                                                                                                                         |
+| `overwrite`        | `bool`                     | When `True`, overwrite files that already exist, without asking.<br>See the [`overwrite`][] setting for related information.                                                                                                                               |
+| `pretend`          | `bool`                     | When `True`, produce no real rendering.<br>See the [`pretend`][] setting for related information.                                                                                                                                                          |
+| `quiet`            | `bool`                     | When `True`, disable all output.<br>See the [`quiet`][] setting for related information.                                                                                                                                                                   |
+| `sep`              | `str`                      | The operating system-specific directory separator.                                                                                                                                                                                                         |
+| `settings`         | `pydantic.BaseModel`       | Specified settings that control copier's behavior.<br>See the [`settings`][] page for related information.                                                                                                                                                 |
```

Comment:
@sisp Thoughts on the content for this one?

Reply:
How about referring to the [`copier.settings.Settings`](https://github.com/copier-org/copier/blob/e2ce0d4f13f439c6d9467fc4bf0e0af5f6a60989/copier/settings.py#L21) model class via `[copier.settings.Settings][]` (thanks to mkdocstrings)?

```suggestion
| `settings`         | [copier.settings.Settings][] | Specified settings that control copier's behavior.<br>See the [`settings`][] page for related information.                                                                                                                                                                                                                                                                                         |
```

(The other rows will need re-formatting because of the wider column ...)

Reply:
Is the problem with `[copier.settings.Settings][]` that it doesn't get rendered as inline code? If so, I imagine ``[`copier.settings.Settings`][]`` or ``[`copier.settings.Settings`][copier.settings.Settings]`` should work – not tested though, as I'm commenting from mobile right now.

Reply:
Yup, _perfect_! 👌 Which do you prefer `Settings` (A)  vs `copier.settings.Settings` (B) as the display value?

A ![SCR-20250607-qqea](https://github.com/user-attachments/assets/47b5019d-5781-4fbf-94c7-584ddada9f2e)

B ![SCR-20250607-qqrn](https://github.com/user-attachments/assets/e5bb238c-72c5-4e5b-bf87-deed7ad9f5c4)

Reply:
I think (B) is better because it's more explicit.

Reply:
Great, since (B) is what's currently in the PR. I'll mark this one resolved then 👍